Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -1218,6 +1218,14 @@ bool AccessibilityRenderObject::computeIsIgnored() const
         if (canSetFocusAttribute())
             return false;
 
+        // https://github.com/w3c/aria/pull/2224
+        // If the image is a descendant of a <figure>, don't ignore it due to <figure> potentially participating in its accname.
+        Node* node = m_renderer->node();
+        for (RefPtr ancestor = node->parentNode(); ancestor; ancestor = ancestor->parentNode()) {
```

Comment:
Missing null-check on this node. This would work:

```cpp
auto* node = m_renderer->node();
for (RefPtr ancestor = node ? node->parentNode() : nullptr; ancestor; ancestor = ancestor->parentNode()) {
```

Raw pointer is OK for `node` since we only use it to call `parentNode`, which is a "trivial" function (inlined read of a member variable).